webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/components/AuditsTable.js

```diff
@@ -4,35 +4,30 @@ import { withRenderHandler } from '../../../../common/HOC';
 import AuditsList from '../../../../components/AuditsList';
 import AuditsLoadingPage from './AuditsLoadingPage';
 import Pagination from '../../../../components/Pagination/Pagination';
-import { usePaginationOptions } from '../../../../Root/Context/ForemanContext';
```

Comment:
Just realized we don't actually need to pass this into the component, as it already defaults to it.